docs/src/development/transfer-dedicated.md

```diff
@@ -51,20 +51,28 @@ Which should give a JSON output containing something like this:
 ]
 ```
 
-The part you want is the user, password, and "path", which means the DB name.
-Ignore the rest.
+The values needed are:
+- `username` of your database (`<username>`).
+- `password` of your database (`<password>`). If empty, remove the flag.
+- "path", which is the name of the database (`<dbname>`).
 
-Now, run the following command on your local computer:
+Now, adapt and run the following command on your local computer:
 
 ```bash
-ssh <USERNAME>@<CLUSTER_NAME>.ent.platform.sh 'mysqldump --single-transaction -u <user> -p<pass> -h localhost <dbname> | gzip' > database.gz
+ssh <USERNAME>@<CLUSTER_NAME>.ent.platform.sh 'mysqldump --single-transaction -u <username> -p<password> -h database.internal <dbname> | gzip' > database.gz
 ```
 
-That runs a `mysqldump` command on the server, compress it using `gzip`,
-and stream the output to a file named `database.gz` on your local computer.
+That runs a `mysqldump` command on the server, compresses it using `gzip`,
+and streams the output to a file named `database.gz` on your local computer.
 
 (If you'd prefer, `bzip2` and `xz` are also available.)
 
+Example with an empty database password:
+```bash
+ssh <USERNAME>@<CLUSTER_NAME>.ent.platform.sh 'mysqldump --single-transaction -u user -h database.internal main | gzip' > database.gz
+```
+
+
```

Comment:
:clipboard: 
```suggestion
Example with an empty database password:

```bash
ssh <USERNAME>@<CLUSTER_NAME>.ent.platform.sh 'mysqldump --single-transaction -u user -h database.internal main | gzip' > database.gz
```

```

:question: If it's going to be an example with real values, I'm not sure I'd keep the variables at the start. But maybe? Not sure if it's more confusing to have it in or out.

:nail_care: The introduction could be more of a full sentence rather than a fragment.

Reply:
I wasn't sure which values I should use for the enterprise cluster SSH. Do we have example values?

Reply:
Here's an example ssh command for a Gen 2 cluster: `ssh 1.ent-ljj2omdt2mmuk-production-vohbr3y@ssh.eu-3.platform.sh`

Reply:
See the context for an example SSH command for a Gen 2 cluster